Azure DevOps.md

```diff
@@ -0,0 +1,34 @@
+# Azure Dev Ops
+
+
+**User Guide**
+
+ - Microsoft has an extensive library available to guide users on their application lifecycle management platform, Azure DevOps:
+
+ - [https://docs.microsoft.com/en-us/azure/devops/user-guide/](https://docs.microsoft.com/en-us/azure/devops/user-guide/)
+
+**Training**
+
+ - Microsoft Learn has free training available to help alleviate culture shock:
+
+ - [Evolve your DevOps practices](https://docs.microsoft.com/en-us/learn/paths/evolve-your-devops-practices)
+
+ - [Build applications with Azure DevOps](https://docs.microsoft.com/en-us/learn/paths/build-applications-with-azure-devops/)
+
+ - Through your MSDN subscription, you may have access to PluralSight and their courses:
+
+ - [Continuous Delivery and DevOps With Azure DevOps](https://www.pluralsight.com/paths/continuous-delivery-and-devops-with-azure-devops)
+
+**Documentation**
+
+ - [https://docs.microsoft.com/en-ca/azure/devops/?view=azure-devops](https://docs.microsoft.com/en-ca/azure/devops/?view=azure-devops)
```

Comment:
This should probably get added to https://esdc-devcop.github.io/ under Refreces -> More

Reply:
[Done](https://github.com/esdc-devcop/esdc-devcop.github.io/pull/101)